ack/readers/awin_advertiser/cli.py

```diff
@@ -31,7 +32,7 @@
 @click.option("--awin-campaign", default=None)
 @click.option("--awin-timezone", required=True)
 @click.option("--awin-interval", default=None)
-@click.option("--awin-remove-tags", default=False)
+@click.option("--awin-remove-tags", default=False, type=Boolean)
```

Comment:
You should use the type that is build with Click: `type=click.BOOL` and not import a boolean from another library ;)

Reply:
ahaha I didn't even see that it did that